be/src/Project/Web/Seller/Seller.Web/Areas/Products/ModelBuilders/ProductCardFormModelBuilder.cs

```diff
@@ -57,13 +59,21 @@ public async Task<ProductCardFormViewModel> BuildModelAsync(ComponentModelBase c
 
             if (componentModel.Id.HasValue)
             {
-                var categorySchema = await _categoriesRepository.GetCategorySchemaAsync(componentModel.Token, componentModel.Language, componentModel.Id);
+                var categorySchemas = await _categoriesRepository.GetCategorySchemasAsync(componentModel.Token, componentModel.Language, componentModel.Id);
 
-                if (categorySchema is not null)
+                if (categorySchemas is not null)
                 {
-                    viewModel.Id = componentModel.Id;
-                    viewModel.Schema = categorySchema.Schema;
-                    viewModel.UiSchema = categorySchema.UiSchema;
+                    viewModel.Id = categorySchemas.Id;
```

Comment:
**suggestion:** Instead of the id of the scheme, there should be the id of the category, i.e. componentModel.Id and the passing of this id should be before the condition, because if the id is not passed, we will not be able to create a scheme when it is empty